github-actions/check-data-assets/README.md

```diff
@@ -1,10 +1,23 @@
-# check-data-assets
+# Gable: Check Data Assets
 
 Checks the data assets specified in the command against their contracts in Gable.
 
 This action is a thin wrapper around the `gable` [Python CLI](https://pypi.org/project/gable/)
 
-```
+## Inputs
+
+| parameter | description | required | default |
+| --- | --- | --- | --- |
+| gable-api-endpoint | Gable API Endpoint in the format https://api.<organization>.gabledata.com | `true` |  |
```

Comment:
seems like `organization` didn't get populated correctly in the markdown doc: 
<img width="490" alt="image" src="https://github.com/gabledata/cicd/assets/20962570/10c6e461-45bc-4d13-abe6-fbfcfd6aad15">

Reply:
good catch! I should have rendered all this markdown to verify myself before putting the PR up.